docs/manage/partitions/flex/create-edit-partitions-flex.md

```diff
@@ -25,6 +25,10 @@ To create or edit a Partition, you must be an account Administrator or have th
 1. Click **+ Add Partition**.
 1. The **Create New Partition** pane appears.<br/><img src={useBaseUrl('img/partitions-data-tiers/create-new-partition-flex.png')} alt="create-new-partition-flex.png"  style={{border:'1px solid gray'}} width="300"/>
 1. **Name**. Enter a name for the Partition. Partitions must be named alphanumerically, with no special characters, with the exception of underscores (`_`). However, a Partition name cannot start with `sumologic_` or an underscore `_`.
+1. (Optional) **Include this partition in default scope**. This checkbox is already selected. Deselect this checkbox if you need to exclude this partition as the default scope in your search.
```

Comment:
Along with this we should explain to the customers what does default scope mean as it is a new concept that we are introducing to them